server/cmd/mmctl/commands/channel.go

```diff
@@ -612,6 +613,15 @@ func getPrivateChannels(c client.Client, teamID string) ([]*model.Channel, error
 }
 
 func deleteChannelsCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	app, err := commands.InitDBCommandContextCobra(cmd)
+	if err != nil {
+		printer.PrintT("App initialization failed: %s", err.Error())
+		return err
+	}
+	deleteEnabled := app.Config().ServiceSettings.EnableAPIChannelDeletion
```

Comment:
That only works if `mmctl` is run on the same server as MM. Let' use `c.GetConfig` instead.